readthedocs/docsitalia/settings/docker.py

```diff
@@ -0,0 +1,259 @@
+    }
+
+    ACCOUNT_DEFAULT_HTTP_PROTOCOL = os.environ.get('ACCOUNT_HTTP_PROTO', 'http')
+
+    ADMINS = (
+        ('Test', 'test@{{ rtd_domain }}'),
+    )
+    TIME_ZONE = 'Europe/Rome'
+    LANGUAGE_CODE = 'it-it'
+
+    CORS_ORIGIN_WHITELIST = (
+        '%s:8000' % os.environ['RTD_DOMAIN'],
+    )
+
+    if os.environ.get('CORS_HEADERS_HOSTS', False) == 'all':
+        CORS_ORIGIN_ALLOW_ALL = True
+
+    WEBSOCKET_HOST = '%s:8088' % os.environ['RTD_DOMAIN'],
+
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql_psycopg2',
+            'NAME': 'rtd',
+            'USER': 'docs',
+            'PASSWORD': 'docs',
+            'HOST': 'db',
+            'PORT': '5432',
+        },
+    }
+
+    # Etc
+    RESTRUCTUREDTEXT_FILTER_SETTINGS = {
+        'cloak_email_addresses': True,
+        'file_insertion_enabled': False,
+        'raw_enabled': False,
+        'strip_comments': True,
+        'doctitle_xform': True,
+        'sectsubtitle_xform': True,
+        'initial_header_level': 2,
+        'report_level': 5,
+        'syntax_highlight': 'none',
+        'math_output': 'latex',
+        'field_name_limit': 50,
+    }
+    USE_PROMOS = False
+
+    ALLOWED_HOSTS = ['*']
+    USER_MATURITY_DAYS = 14
+    READTHEDOCSEXT_MONITORING_CACHE = 'stats'
+    DEFAULT_VERSION_PRIVACY_LEVEL = os.environ['DEFAULT_VERSION_PRIVACY_LEVEL']
+
+    @property
+    def TEXTCLASSIFIER_DATA_FILE(self):
+        return os.path.join(self.SITE_ROOT, 'textclassifier.json')
+
+    # Banned" projects
+    HTML_ONLY_PROJECTS = (
+        'atom',
+        'galaxy-central',
+        'django-geoposition',
+    )
+
+    # Add fancy sessions after the session middleware
+    @property
+    def MIDDLEWARE_CLASSES(self):
+        classes = super(CommunityProdSettings, self).MIDDLEWARE_CLASSES
+        classes = list(classes)
+        index = classes.index(
+            'readthedocs.core.middleware.FooterNoSessionMiddleware'
+        )
+        classes.insert(
+            index + 1,
+            'restrictedsessions.middleware.RestrictedSessionsMiddleware'
+        )
+        return tuple(classes)
+
+    RESTRICTEDSESSIONS_AUTHED_ONLY = True
+
+    # Logging
+    @property
+    def LOGGING(self):
+        logging = super(CommunityProdSettings, self).LOGGING
+        logging['formatters']['syslog'] = {
+            'format': 'readthedocs/%(name)s[%(process)d]: %(levelname)s %(message)s [%(name)s:%(lineno)s]',
+            'datefmt': '%d/%b/%Y %H:%M:%S'
+        }
+        logging['loggers'] = {
+            '': {
+                'handlers': ['console'],
+                'level': 'INFO',
+            }
+        }
+        return logging
+
+
+CommunityProdSettings.load_settings(__name__)
+
+if not os.environ.get('DJANGO_SETTINGS_SKIP_LOCAL', False):
+    try:
+        # pylint: disable=unused-wildcard-import
+        from .local_settings import *  # noqa
+    except ImportError:
+        pass
```

Comment:
Nota di carattere generale: commentiamo molto di più i blocchi di variabili, in modo che sia chiaro a chi ci dovrà lavorare in futuro gli scopi di ciascuna di esse.

Reply:
approvo
quel file di settings va completamente riscritto e documentato